src/modules/behavior_controller.cpp

```diff
@@ -81,7 +81,10 @@ namespace BehaviorController
                     // if (BatteryController::getCurrentLevel() > BATT_TOO_LOW_LEVEL) {
                         NRF_LOG_INFO("Triggering a HelloGoodbye Condition");
```

Comment:
Actually I like that you had the alternate log_info indicating we skipped the hellogoobye!

Reply:
I see. I can move that inside the conditional then.

Reply:
Done!